### DISCUSSION TITLE
Render to Figma

### DISCUSSION THREAD

**Discussion by vladchatware** [gluestack/gluestack-ui#1705] [Q&A] [answered]
Hi, I was checking figma template and noticed that it has "auto-generated" in description. 
I went to check the corresponding script or something that gives me the idea how it was done.
I know that there is react-figma package that allows to do just that although it uses react-primitives.

So my question is, doest webpack resolves the alias to react and does the trick or there some more magic involved?

**Comment by ankit-tailor** [comment, marked as answer]
Hey @jellyninjadev, This is generated using our in-house figma plugin.
